app/src/main/java/com/infomaniak/mail/workers/DraftsActionsWorker.kt

```diff
@@ -280,14 +280,16 @@ class DraftsActionsWorker @AssistedInject constructor(
 
         attachmentsToUpload.forEach { attachment ->
             runCatching {
-                attachment.startUpload(localUuid, attachmentsUris)
+                attachment.startUpload(localUuid)
             }.onFailure { exception ->
                 SentryLog.d(TAG, "${exception.message}", exception)
                 if ((exception as Exception).isNetworkException()) throw ApiController.NetworkException()
                 throw exception
             }
         }
 
+        LocalStorageUtils.deleteDraftDir(applicationContext, localUuid, userId, mailbox.mailboxId)
```

Comment:
Is this call really useful ? 
It seems not, as startUpload already call deleteDraftDir

Reply:
True.